packages/connect/src/ExtensionProvider/ExtensionProvider.ts

```diff
@@ -15,9 +15,19 @@ import { logger } from '@polkadot/util';
 import EventEmitter from 'eventemitter3';
 import { isUndefined } from '../utils';
 
-const EXTENSION_ORIGIN = 'extension-provider';
+const origins = {
```

Comment:
For constants use variables instead of object.

Reply:
Fine by me but out of interest .. why?

Reply:
Because it is a constant.. it should not change later in the code. When you add it in an object key, you can alter it and that makes it lose the value of being a constant.
e.g. 
```
const a = '1';
a = '2';
->> Uncaught TypeError: Assignment to constant variable.
-----------
const a = { b: '1' }; // console.log(a) -> {b: '1'}
a.b = '2'; // console.log(a) -> {b: '2'}
```

Reply:
Ah yeah good point!